slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/JdbcTypeTest.scala

```diff
@@ -313,6 +316,20 @@ class JdbcTypeTest extends AsyncTest[JdbcTestDB] {
       () => randomLocalDateTime().toInstant(ZoneOffset.UTC)
     )
 
+  def testPostgresInstantWithTimeZone: Future[Unit] = if (tdb.confName == "postgres") {
+    val withTimeZone = new PostgresProfile.columnTypes.InstantJdbcType {
+      override def sqlTypeName(sym: Option[FieldSymbol]) = "TIMESTAMPTZ"
```

Comment:
Maybe add a comment in here as to why this test is needed and specific to postgres, to give it some context?
Something like "Testing for an Instant mapping for Postgres that uses the SQL time TIMESTAMPTZ rather than the default SQL mapping that is TIMESTAMP".